app/views/layouts/govuk_publishing_components/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <%= csrf_meta_tags %>
     <%= favicon_link_tag "govuk_publishing_components/favicon-production.png" %>
 
-    <%= stylesheet_link_tag "component_guide/application", media: "screen" %>
+    <%= stylesheet_link_tag "component_guide/application", media: "all" %>
```

Comment:
Is it worth putting this change in a separate commit? It's related, but it feels like a significant thing all of its own.

Reply:
Yep good idea. I'll remove it from here.